.chloggen/fix_clickhouse-exporter-log-service-name.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: clickhouseexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Fix incorrect ServiceName set for Logs Records
```

nit: maybe mention `service.name` since that's user facing message
```suggestion
note: Fix incorrect service.name set for Logs Records
```

---

I updated changelog title to reflect what exactly was fixed